pages/help/glossary.mdx

```diff
@@ -0,0 +1,91 @@
+## General 
+
+| Acronym/Abbreviation         | Definition                                                                               |
+| ---------------------------- | ---------------------------------------------------------------------------------------- |
+| ACL                          | Access Control List                                                                      |
+| API                          | Application Programming Interface                                                        |
```

Comment:
I think API, CI/CD, DCA, HTTP, REST, URL, and UUID can be moved to a "Web Applications" section at the very bottom.